manifest.json

```diff
@@ -1401,6 +1401,9 @@
             "authors": {
                 "art0007i": {
                     "url": "https://github.com/art0007i"
+                },
+                "Banane9": {
```

Comment:
It is up to @art0007i if they'd like to add you as an author on their mod. You can wait for a response from them or remove this change and make it a separate PR so that this one can proceed to be merged.

Reply:
yeah im cool with it

Reply:
Alright